api4/user.go

```diff
@@ -965,6 +975,11 @@ func login(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	user.Sanitize(map[string]bool{})
 
+	if extensionId != "" {
+		w.Header().Set("Access-Control-Expose-Headers", "Token")
```

Comment:
this is not safe, this is exposing the user token in a Header that can be intercepted, why not taking the token from the cookies that are set on login?

Reply:
Agreed, but since chrome makes cross-origin calls it's not possible to authenticate without this in the headers.  If there's another way then please let me know.  What we could do is utilize the chrome communication api to send over the data like we do with SAML.   Not sure if that's the ideal solution though